package-scripts.js

```diff
@@ -64,11 +64,14 @@ module.exports = {
     dev: {
       script: series([
         startInNewWindow('npm start dev.mongo --silent'),
-        startInNewWindow('cross-env PORT=8080 npm start dev.client --silent'),
+        startInNewWindow(
+          './node_modules/.bin/cross-env PORT=8080 ' +
+            'npm start dev.client --silent'
```

Comment:
Oh, I should have mentioned, use `oneLine` from [`common-tags`](https://github.com/declandewet/common-tags)! Woo!